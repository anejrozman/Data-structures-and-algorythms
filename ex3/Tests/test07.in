16,43,4
5.945935059812619,1.5652133513884037,5.421927129057276,4.717596191529969,7.371178235374899,9.783887999213222,4.9766324965106445,8.938522500316566,2.7465959834995095,3.1405727835447665,3.663405982875677,7.8732023344825715,5.412678182356878,3.7583961309483076,2.8711618105346464,4.423965975691447
4,1,768.942
12,3,6.945
3,1,646.94
9,15,11.301
13,5,909.521
6,1,146.017
15,10,660.579
0,10,532.094
8,2,406.173
12,11,679.052
11,9,296.438
7,1,683.847
6,13,171.675
15,12,173.709
9,5,173.772
11,15,177.99
5,15,178.288
9,14,181.302
6,5,54.493
4,12,570.602
11,2,59.02
13,0,700.127
0,7,323.064
1,8,323.071
4,2,838.575
0,6,713.023
0,4,203.548
5,10,593.238
0,14,82.835
2,10,211.097
13,2,215.607
11,0,345.724
14,12,96.806
6,4,353.828
6,14,227.14
2,7,236.845
0,15,364.221
14,10,115.118
10,3,244.606
7,13,630.906
2,0,757.817
8,4,372.946
3,8,127.942
