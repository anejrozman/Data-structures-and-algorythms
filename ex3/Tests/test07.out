12,1263.5121
2,42.6879
1,0
1,0
